Swamp Sneak/Assets/Scripts/EnemyLineOfSight.cs

```diff
@@ -71,4 +71,27 @@ public class EnemyLineOfSight : MonoBehaviour {
 	public void SetSightDistance(int sd) {
 		sightDistance = sd;
 	}
+	
+	bool stealthCheck()
+	{
+
+		//Function to check if player is stealthed and also handle RNG factor
+		if(GameObject.Find("Player").GetComponent<ThirdPersonCharacter>().getStealth() == false)
```

Comment:
These calls are very inefficient. These `Find`s are running every frame. Also, they should really be using [FindByTag()](https://docs.unity3d.com/ScriptReference/GameObject.FindWithTag.html) since we explicitly put tags on the gamecontrol and player for that reason.

Though since we are the end of the sprint, it may not be time effective to fix these if the Demo is at least in a working state. This comment is more as an FYI :)